tests/test_external_search.py

```diff
@@ -3194,7 +3013,7 @@ def test_next_page(self):
 
 class TestBulkUpdate(DatabaseTest):
 
-    def test_works_not_presentation_ready_removed_from_index(self):
+    def test_works_not_presentation_ready_kept_in_index(self):
```

Comment:
This is the test of the incidental change I made in this branch, since we now keep works in the index even if they're not presentation-ready.